test/e2e/scheduler_predicates.go

```diff
@@ -262,11 +262,18 @@ var _ = framework.KubeDescribe("SchedulerPredicates [Serial]", func() {
 	// It assumes that cluster add-on pods stay stable and cannot be run in parallel with any other test that touches Nodes or Pods.
 	// It is so because we need to have precise control on what's running in the cluster.
 	It("validates resource limits of pods that are allowed to run [Conformance]", func() {
+		MAX_NUMBER_OF_PODS := int64(10)
```

Comment:
Define as a constant and use camel case names (start from a small letter, as we don't want it to be exported)